docs/MULTIREPO.md

```diff
@@ -0,0 +1,20 @@
+# This doc describes a workflow for using multiple repositories in a development flow.
+
+## Problem
+For some of the assets we (will) use in the app we do not have the rights to freely distribute them. GitHub's ToS specify that anything in the public repo must be freely redistributable.
```

Comment:
I don’t think giving a legal interpretation of the ToS is something we non-lawyers should give.  Suffice to say we have assets that we do not wish to distribute in the public repo.

Reply:
It's not that we do not wish it.. I think not clarifying why is bad from a transparency perspective. I'd be happy to have a different phrasing or approval from a lawyer though..